README.md

```diff
@@ -89,6 +89,7 @@ Allowed options per language:
 
 - `name` (mandatory): the name that should be displayed in the [mkdocs-material language switcher](#using-mkdocs-material-site-language-selector)
 - `link` (default: `./<language>`): the link path to be used in the [mkdocs-material language switcher](#using-mkdocs-material-site-language-selector)
+- `fixed_link` (default: `None`): if specified, when switching to this language, you will be redirected to this link.
```

Comment:
please drop the dot at the end of the sentence, I don't use them on the rest of the items listing here

Reply:
Solved with [111e538932a119cd9924eef11b7f89b84f95f446](https://github.com/dariocurr/mkdocs-static-i18n/commit/111e538932a119cd9924eef11b7f89b84f95f446)